### PR TITLE
frontend: WTForms validate function now expect additional parameter

### LINKED ISSUE #2482: CI fails for Fedora rawhide
In Rawhide (F38) there is  `python3-flask-wtf-1.1.0-1.fc38.noarch` which causes our tests to fail. Downgrading to `python3-flask-wtf-1.0.1-3.fc38.noarch` fixes it.

We should see what changes happened in the new version and update our code.

### PR DESCRIPTION
**Pull request by FrostyX**
Fix #2482
See https://github.com/wtforms/flask-wtf/pull/479

### CODE REVIEW

File: frontend/coprs_frontend/coprs/forms.py [hunk 1]
```diff
@@ -735,8 +735,8 @@ class F(CoprForm):
             def selected_chroots(self):
                 return self.chroots.data
 
-            def validate(self):
-                if not super(F, self).validate():
+            def validate(self, extra_validators=None):
+                if not super(F, self).validate(extra_validators):
```
Comment (praiskup) [inline frontend/coprs_frontend/coprs/forms.py:739]: Considering the old WTForms (our current deployment where the argument doesn't exist, yet) -> is it safe to pass None here?  I'm afraid it is some other argument actually.

**Comment by praiskup** [comment]
Nice, thank you for quickly fixing this!

File: frontend/coprs_frontend/coprs/forms.py (hunk 1 above)
Comment (FrostyX) [reply frontend/coprs_frontend/coprs/forms.py:739]: I know what you mean, I was a bit concerned as well. But I just checked

The `validate` method is not defined in `python3-flask-wtf`. The class hierarchy is funny.

    F < CoprForm < BaseForm < FlaskForm < Form < BaseForm
    These are ours          | flask-wtf |     wtforms

The `validate` method is defined in `BaseForm` in `python3-wtforms`, and it already takes `extra_validators`.

```
[root@copr-fe ~][PROD]# cat /usr/lib/python3.11/site-packages/wtforms/form.py |grep "def validate"
    def validate(self, extra_validators=None):
    def validate(self, extra_validators=None):
```

Comment (praiskup) [reply frontend/coprs_frontend/coprs/forms.py:739]: so can we do "extra_validators=extra_validators" instead, just to be sure?

Comment (FrostyX) [reply frontend/coprs_frontend/coprs/forms.py]: Good idea, done.

**Review by praiskup — APPROVED** [review]

**Pull request merged**